test/e2e/tls_test.go

```diff
@@ -39,23 +39,23 @@ var _ = Describe("TLS activation - application", Ordered, func() {
 
 		By("create a service account for OLS user")
 		cleanUp, err := client.CreateServiceAccount(OLSNameSpace, testSAName)
-		Expect(err).NotTo(HaveOccurred())
 		cleanUpFuncs = append(cleanUpFuncs, cleanUp)
```

Comment:
does this order change help? if there is an error, the cleanUp function is `nil` that cannot be referred as a function to execute in the `afterAll` function.

Reply:
@raptorsun you are right. It does nto make sense at all. I reverted it. Thanks.